pages/linux/grafana-cli.md

```diff
@@ -0,0 +1,16 @@
+# grafana-cli
+
+> A small executable that is bundled with Grafana serve.
+> More information: <https://grafana.com>.
+
+- Install a grafana plugin:
+
+`grafana-cli plugins install "{{plugin-id}}"`
```

Comment:
```suggestion
`grafana-cli plugins {{install|update|remove}} {{plugin_id1 plugin_id2 ...}}`
```

If you know that plugin id doesn't contain spaces in it or documentation states it explicitly please remove quotes.